service/hub/deploy/manifests/deploy.yaml

```diff
@@ -21,13 +27,108 @@ data:
   tls.key: |
     -----BEGIN RSA PRIVATE KEY-----
     -----END RSA PRIVATE KEY-----
-kind: ConfigMap
+---
```

Comment:
Split Sa rbac config to another yaml file to better maintain upgrade/install scripts?